apps/web/vibes/soul/primitives/navigation/index.tsx

```diff
@@ -218,11 +220,11 @@ export const Navigation = forwardRef(function Navigation(
   return (
     <div {...rest} className="sticky top-0 z-30 w-full @container">
       <div
-        ref={ref}
         onMouseLeave={() => setNavOpen(false)}
         className="relative mx-auto w-full max-w-screen-2xl px-0 text-foreground @4xl:px-2 @4xl:pt-2"
       >
         <nav
+          ref={ref}
```

Comment:
Moving `ref` to point to the top interactive element from user POV. Among other things, this allows Makeswift builder to accurately show component boundaries when editing the navbar.